modules/tinymce/src/plugins/quickbars/main/ts/selection/Toolbars.ts

```diff
@@ -1,11 +1,18 @@
+import { Class, SugarElement } from '@ephox/sugar';
+
 import Editor from 'tinymce/core/api/Editor';
 
 import * as Options from '../api/Options';
 
 const addToEditor = (editor: Editor): void => {
   const isEditable = (node: Element | null): boolean => editor.dom.isEditable(node);
   const isInEditableContext = (el: Element) => isEditable(el.parentElement);
-  const isImage = (node: Element): boolean => node.nodeName === 'IMG' || node.nodeName === 'FIGURE' && /image/i.test(node.className) && isInEditableContext(node);
+  const isImage = (node: Element): boolean => {
+    const isImageFigure = node.nodeName === 'FIGURE' && /image/i.test(node.className);
+    const isImage = node.nodeName === 'IMG' || isImageFigure;
+    const isPagebreak = Class.has(SugarElement.fromDom(node), 'mce-pagebreak');
```

Comment:
I wonder if we could reach across plugins and import this string directly at compile time 🤔 
https://github.com/tinymce/tinymce/blob/9c486f7c951e55ae34bb548efe897eec705ac04d/modules/tinymce/src/plugins/pagebreak/main/ts/core/FilterContent.ts#L6

Reply:
Could? Yes. Should? More debatable.